raspbian11: Use signed-by style on GPG key for apt repo

cosmo0920 opened the pull request.

cosmo0920 committed:
raspbian11: Use signed-by style on GPG key for apt repo

Signed-off-by: Hiroshi Hatake <hatake@calyptia.com>

patrick-stephens commented on installation/linux/raspbian-raspberry-pi.md:
```diff
@@ -5,14 +5,20 @@ Fluent Bit is distributed as **fluent-bit** package and is available for the Ras
 * Raspbian Buster \(10\)
 
-## Server GPG key
+## Server GPG key (old style)
```
I'd say this is confusing, let's just do it one way - the old docs are always around if people want the old way.

cosmo0920 replied:
Yup. I understand. Personally, I wanted to add common section to handle apt-key deprecation.

patrick-stephens reviewed (commented):
Let's only have one way to do it.

Could you update the other Debian targets as well @cosmo0920 to be consistent?
@ehienabs can we have a common section inserted rather than duplicating it everywhere?

eschabell committed:
docs: installation: linux: raspbian: fix GPG key sections and style issues

  - Add deprecation warning for apt-key (old style) pointing to new style section
  - Replace sudo bash redirect with sudo tee pipeline in new style command
  - Replace keyrings directory reference with explicit path
  - Fix vale failures for many phrases in the text

  Applies to #1147

Signed-off-by: Eric D. Schabell <eric@schabell.org>
